src/datagrid/DataGrid.tsx

```diff
@@ -43,6 +43,7 @@ import {DataGridColumnResize} from "./DataGridColumnResize";
  * @param {selectedRowCount} number of selected rows across all pages
  * @param {dataqa} quality engineering tag
  * @param {id} unique ID for datagrid
+ * @param {isLoading} if true shows loading spinner else shows datagrid
```

Comment:
Comment can be updated, its only showing loadingSpinner on a single row, that too at the CheckBox's place. 

I have some questions 

1) if one roo is in expanded state,  backend call is going on and if user expands another row then do we keep both rows in expanded state.  (We should I think)

2) If the expanded row makes backend call, which takes longer time to return.  where do we show error messgaes and how the Expanded rows behave in this case.

Reply:
@maheshkulkarni4  
**For Question 1 :** If user expands more than one row we are keeping all rows in expanded state until user collapse it. This behaviour is as per VMware clarity.

**For Question 2 :** In case of error Parent component who is sending callback should send error to datagrid and datagrid can show that error in expandable row section. Vmware clarity do not have any such use case as of now. But we can support it.